index.html

```diff
@@ -133,37 +139,37 @@ <h2 class="headings heading-">Featured Speakers</h2>
         </section>
```

Comment:
## Optional

- [ ] Currently, there is a break in your code where you have the "more" div (the last div in this section) kindly fix this for the mobile version
![image](https://user-images.githubusercontent.com/69638013/157533269-bc238a33-cc77-4454-b300-61a0f9d9763a.png)